WebRoot/templates/share/Dialogs.template

```diff
@@ -722,6 +722,65 @@
 	</div>
 </template>
 
+<template id='share.Dialogs#ZmTwoFactorCustomLoginPage'>
+		<div class="ZmTwoFactorCustomSetupLoginPage">		
```

Comment:
Are we missing a DIV here or its just an indentation issue ?

Reply:
yes had missed a div , yesterday had mad a minor change on a review suggestion